### PR TITLE
Build up models

### PR DESCRIPTION
**Pull request by ruiccarvalho**
After merging #12 and having a basic set of tables with the right columns, constraints, and referential integrity (foreign_keys), the basic ActiveRecord/ActiveModel logic for the corresponding classes must be put in place.

### CODE REVIEW

File: app/models/country.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class Country < ApplicationRecord
+  validates :code, presence: true, uniqueness: true
+  validates :name, presence: true, uniqueness: true
```
Comment (gdiasdasilva) [inline app/models/country.rb:3]: Do we need this at the AR level? 

If we'll have a predefined set of countries, loaded through some script and not manageable through the app, maybe it should be a constraint at the database level, if needed.

File: app/models/movie.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Movie < ApplicationRecord
+  has_many :directions
+  has_many :directors, through: :directions
+  validates :original_title, presence: true
+  validates :directors, presence: true
```
Comment (gdiasdasilva) [inline app/models/movie.rb:5]: Does this work? 👀 

If not, maybe we should try [validates_length_of](https://apidock.com/rails/v6.0.0/ActiveRecord/Validations/ClassMethods/validates_length_of).

File: app/models/team.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Team < ApplicationRecord
+  has_many :memberships
+  has_many :rounds
+  has_many :users, through: :memberships
+  validates :name, presence: true, length: { minimum: 5, maximum: 40 }
```
Comment (gdiasdasilva) [inline app/models/team.rb:5]: Poor Kiko 😞

File: spec/models/round_spec.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
```
Comment (gdiasdasilva) [inline spec/models/round_spec.rb]: There is some overall inconsistency with quotes. I'd suggest we set double quotes as the standard, if that's ok to you. 

We can apply rubocop's autofix for this in the future.

File: app/models/country.rb (hunk 1 above)
Comment (ruiccarvalho) [reply app/models/country.rb:3]: Well, neither are **necessary**. But is there a good reason not to put them? Note, btw, that they're being imported one-by-one in AR — not the most efficient way, but the easiest and more legible, I think, and efficiency is not a problem for something that runs once in a lifetime.
In any case, I've also put the constraints at the db level.

File: app/models/movie.rb (hunk 2 above)
Comment (ruiccarvalho) [reply app/models/movie.rb:5]: It most certainly does! 🤓

File: spec/models/round_spec.rb (hunk 4 above)
Comment (ruiccarvalho) [reply spec/models/round_spec.rb]: Yes, right now is inconsistent with any option, let's discuss. But maybe we could merge and change everywhere in another PR?

**Pull request merged**